app/channels/application_cable/connection.rb

```diff
@@ -3,8 +3,8 @@ class Connection < ActionCable::Connection::Base
     identified_by :current_user
 
     def connect
-      #self.current_user = nil
-      token = cookies.encrypted['SOME SECRET']['token']
+      # self.current_user = nil
+      token = cookies.encrypted['_samples_extraction_session']['token']
```

Comment:
The update to rack broke the previous session cookie key, as cookie values no longer get unescaped due to a CVE, so the space causes havoc. This reverts to the default cookie name.

The previous name 'SOME SECRET' implies that this key should remain secret (and presumably hints at a more secure key injected during deployment). However there is no reason to keep this secret, and this is easily publicly available to anyone inspecting their cookies.